src/main/java/pro/sky/StreamAPIAndOptional/DepartmentsService.java

```diff
@@ -0,0 +1,12 @@
+package pro.sky.StreamAPIAndOptional;
+
+import java.util.List;
+import java.util.Map;
+
+public interface DepartmentsService {
```

Comment:
```suggestion
public interface DepartmentService {
```
в единственном числе называем классы